app/src/main/java/co/nimblehq/compose/crypto/ui/screens/information/CoinInfoAppbar.kt

```diff
@@ -0,0 +1,79 @@
+package co.nimblehq.compose.crypto.ui.screens.information
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.*
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.tooling.preview.Preview
+import co.nimblehq.compose.crypto.R
+import co.nimblehq.compose.crypto.ui.theme.ComposeTheme
+import co.nimblehq.compose.crypto.ui.theme.Style
+import co.nimblehq.compose.crypto.ui.theme.Style.textColor
+
+@Suppress("FunctionNaming", "LongMethod")
+@Composable
+fun CoinInfoAppbar(
```

Comment:
> Could we apply built-in TopAppBar supported to handle the appbar?

@hoangnguyen92dn Yeah, We can use `TopAppBar,` but we need to use `Scaffold` as well. I have tried with `TopAppBar` too.  It seems like we need to update the theme to match the design because it's a material design. That's why I think the current way is easier 🙏 

>And with the UX, when users scroll up/down, the appbar should always stick on the top instead

Yup, I will update for this 🚀

Reply:
> @hoangnguyen92dn Yeah, We can use TopAppBar, but we need to use Scaffold as well. I have tried with TopAppBar too. It seems like we need to update the theme to match the design because it's a material design. That's why I think the current way is easier 🙏

That's fine na. I mean that we should try as much as possible all the new components to practice 😄 It's the chance for us to do that. Native components could handle many cases and sometimes we don't need to reinvent the wheels.

Reply:
@Wadeewee How much effort you have to spend for that experiment?

Reply:
@luongvo If we use `TopAppBar,` We need to update the theme's dependencies, e.g. (`shadow,` `evaluation,` It would be impacted with `style.xml`). Also, we need to update the layout to support the scrolling inside `Scaffold,` including the bottom sticky, So I think the custom one is less handle and is more flexible. cc @hoangnguyen92dn 

If you guys prefer to use `TopAppBar,` I'm good, but please allow me to implement it in the next PR. I will create chore PR for this issue later 🙇‍♀️

Reply:
That's a fair point @Wadeewee. I think we could handle it later in a new feature task @hoangnguyen92dn 🤔